ArduCopter/motors.pde

```diff
@@ -191,9 +173,6 @@ static bool init_arm_motors(bool arming_from_gcs)
     // enable output to motors
     output_min();
 
-    // finally actually arm the motors
-    motors.armed(true);
-
     // log arming to dataflash
     Log_Write_Event(DATA_ARMED);
```

Comment:
Maybe we want to split this to two events now (or just move that one to the arm_check) otherwise there will be a two seconds delay between the log event and the actual arm time